terasoluna-gfw-common-libraries/terasoluna-gfw-web/src/main/java/org/terasoluna/gfw/web/logging/TraceLoggingInterceptor.java

```diff
@@ -101,19 +101,18 @@ public class TraceLoggingInterceptor implements HandlerInterceptor {
     @Override
     public boolean preHandle(HttpServletRequest request,
             HttpServletResponse response, Object handler) {
-        if (!(handler instanceof HandlerMethod)) {
-            return true;
+        if (handler instanceof HandlerMethod) {
+            if (logger.isTraceEnabled()) {
+                HandlerMethod handlerMethod = (HandlerMethod) handler;
```

Comment:
- [ ] Use instanceof pattern matching.

Reply:
f71e3188026a0deb697528b3ce8092eb4068a148
Fixed to use instanceof pattern matching.